### ISSUE TITLE
Perl test suite -- support for a remote target

### ISSUE THREAD

**Issue by oneru** [mrash/fwknop#196]
It could be useful to allow the test suite to target a remote IP for certain actions.  The specific use mentioned today was to generate broken SPA packets in order to stress test a remote SPA implementation.  

This could also be useful for certain platforms where running the full test suite is impractical, OpenWrt for instance.

**Comment by mrash** [comment]
Good idea. I'll do some thinking on this. One quick way to get some benefit would be to send the fuzzing packet set over the network.
